models/User.go

```diff
@@ -834,7 +850,8 @@ func PlaceBidOrder(userId uint32, bid *Bid) (uint32, error) {
 	l.Debugf("Check1: Passed.")
 
 	// Second Check: User should have enough cash
-	var cashLeft = int32(user.Cash) - int32(bid.StockQuantity*bid.Price)
+	orderFee := uint32((ORDER_FEE_PERCENT / 100.0) * float32(bid.StockQuantity*bid.Price))
+	cashLeft := int32(user.Cash) - int32(bid.StockQuantity*bid.Price+orderFee)
```

Comment:
uint32 for cashLeft please. Maintain uniformity.

Reply:
cashLeft is going to be unsigned so both the operands on the right need to be int32.

Reply:
you mean it's going to be _signed_?

Reply:
Yeah sorry my bad. cashLeft can be signed.